disperser/cmd/encoder/config.go

```diff
@@ -28,7 +28,7 @@ type Config struct {
 	EncoderConfig    kzg.KzgConfig
 	LoggerConfig     common.LoggerConfig
 	ServerConfig     *encoder.ServerConfig
-	MetricsConfig    encoder.MetrisConfig
+	MetricsConfig    encoder.MetricsConfig
```

Comment:
is there any reason why ServerConfig is a pointer but MetricsConfig is not? Is ServerConfig being shared with another service? If so, add a comment. Otherwise, I'd prob make all of them use same pattern, just for consistency.